### ISSUE TITLE
Improve Swagger with examples in POST operations

### ISSUE THREAD

**Issue by e-pettersson-ericsson** [eiffel-community/eiffel-remrem-generate#153]
### Description
The Swagger interface of REMReM generate should have proper examples of JSON structures in the POST requests. Right now these endpoints have a big blob of irrelevant JSON which does not help the user prepare a body for the HTTP request. I want to replace the default JSON blob with the correct JSON body. 

This could be achieved for example by defining JSON schemas to show how the JSON object should look like and what fields are required/optional etc. I did a quick google search and found it is possible to define JSON schemas using Swagger,  https://swagger.io/docs/specification/describing-request-body/ 

### Motivation
Easier to start playing around with the REST API of REMReM generate.

### Exemplification
Currently we have: 
![image](https://user-images.githubusercontent.com/38855976/69531136-a4471900-0f73-11ea-9bfa-55e2fe0dd305.png)

But I would like a proper example of accurate JSON which could immediately be copied and used. So the yellow box would contain an example value which is actually useful.

### Benefits
More useful Swagger description of the interface.

### Possible Drawbacks
None that I can think of.

**Comment by ZUKSMIR** [comment]
Magnus , Can please tell exactly which JSON example are you referring to ,in the link which you have provided?

**Comment by magnusbaeck** [comment]
The Usage section of https://eiffel-community.github.io/eiffel-remrem-generate/serviceUsage.html contains a table with all the REST endpoints and the following example for /mp:

```
{
  "msgParams": {
    "meta": {
      # Matches the meta object
    }
  },
  "eventParams": {
    "data": {
      # Matches the data object
    },
    "links": {
      # Matches the links object
    }
  }
}
```
For a first-time user this isn't entirely easy to find, although the recently completed (?) migration from GitHub Pages to plain markdown files in the git should improve the situation quite a bit. I suspect it's something like the above that the issue creator is looking for.

**Comment by e-pettersson-ericsson** [comment]
Yes, @magnusbaeck those kind of examples are what I was referring to when I created the issue. It would be very helpful for users trying out the REST API if there examples ready to use already in the Swagger GUI.

**Comment by Christoffer-Cortes** [comment]
Adding examples to a property does not seem to work atm in springfox-swagger. Its been a known issue for a while as can be seen here: https://github.com/springfox/springfox/issues/2873
There may be a workaround to this or simply a different approach.

**Comment by erjamni-raja** [comment]
We tried with some approaches, but no use.
Perhaps we should wait for springfox 3.0.0 release.

**Comment by SantoshNC68** [comment]
This has been closed without a fix https://github.com/springfox/springfox/issues/2873 so how to proceed further on this ?

**Comment by Christoffer-Cortes** [comment]
This may have been fixed through https://github.com/springfox/springfox/issues/2767 and is part of 3.0.0 so I think this can be investigated again.

**Comment by e-pettersson-ericsson** [comment]
Fixed in PR #249
